src/backend/gulpfile.js

```diff
@@ -1,106 +1,125 @@
   });
 
   gulp.task('eventJs', function() {
-
     return gulp.src([dir + 'map.js', dir + 'scroll.js', dir + 'navbar.js', dir + 'popover.js', dir + 'loklak-fetcher.js', dir + 'tweets.js', dir + 'jquery.lazyload.js'])
-    .pipe(iife({ useStrict : false}))
-    .pipe(concat('event.min.js'))
-    .pipe(babel({ presets: ['es2015'] }))
-    .pipe(uglify().on('error', function(e) {
-        console.log("Error while compiling event.js");
-    }))
-    .pipe(gulp.dest(path + '/js/'));
+      .pipe(iife({useStrict: false}))
+      .pipe(concat('event.min.js'))
+      .pipe(babel({presets: ['es2015']}))
+      .pipe(uglify().on('error', function(e) {
+        console.log('Error while compiling event.js' + e);
+      }))
+      .pipe(gulp.dest(path + '/js/'));
   });
 
   gulp.task('roomsJs', function() {
-
-    return gulp.src([dir + 'social.js', dir + 'scroll.js', dir + 'navbar.js', dir + 'tabs.js', dir + 'jquery.lazyload.js'])
-    .pipe(iife({ useStrict : false}))
-    .pipe(concat('rooms.min.js'))
-    .pipe(babel({ presets: ['es2015'] }))
-    .pipe(uglify().on('error', function(e) {
-        console.log("Error while compiling rooms.js");
-    }))
-    .pipe(gulp.dest(path + '/js/'));
+    return gulp.src([dir + 'social.js', dir + 'scroll.js', dir + 'navbar.js', dir + 'jquery.lazyload.js'])
+      .pipe(iife({useStrict: false}))
+      .pipe(concat('rooms.min.js'))
+      .pipe(babel({presets: ['es2015']}))
+      .pipe(uglify().on('error', function(e) {
+        console.log('Error while compiling rooms.js' + e);
+      }))
+      .pipe(gulp.dest(path + '/js/'));
   });
 
   gulp.task('speakersJs', function() {
-
     return gulp.src([dir + 'social.js', dir + 'scroll.js', dir + 'navbar.js', dir + 'popover.js', dir + 'jquery.lazyload.js'])
-    .pipe(iife({ useStrict : false}))
-    .pipe(concat('speakers.min.js'))
-    .pipe(babel({ presets: ['es2015'] }))
-    .pipe(uglify().on('error', function(e) {
-        console.log("Error while compiling speakers.js");
-    }))
-    .pipe(gulp.dest(path + '/js/'));
+      .pipe(iife({useStrict: false}))
+      .pipe(concat('speakers.min.js'))
+      .pipe(babel({presets: ['es2015']}))
+      .pipe(uglify().on('error', function(e) {
+        console.log('Error while compiling speakers.js' + e);
+      }))
+      .pipe(gulp.dest(path + '/js/'));
+  });
+
+  gulp.task('sessionJs', function() {
+    return gulp.src([dir + 'session.js', dir + 'social.js'])
+      .pipe(iife({useStrict: false}))
+      .pipe(concat('session.min.js'))
+      .pipe(babel({presets: ['es2015']}))
+      .pipe(uglify().on('error', function(e) {
+        console.log('Error while compiling session.js' + e);
+      }))
+      .pipe(gulp.dest(path + '/js/'));
+  });
+  gulp.task('mainJs', function() {
+    return gulp.src([dir + 'main.js'])
+      .pipe(iife({useStrict: false}))
+      .pipe(concat('main.min.js'))
+      .pipe(babel({presets: ['es2015']}))
+      .pipe(uglify().on('error', function(e) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines. (padded-blocks)](https://app.codacy.com/app/triplez/open-event-webapp/pullRequest?prid=3226522)